application/shell/openvoiceshellplugin.cpp

```
@@ -0,0 +1,45 @@
+ */
+
+#include "openvoiceshellplugin.h"
+#include "placesmodel.h"
+#include "configuration.h"
+#include <QQmlContext>
+#include <QQmlEngine>
+
+static QObject *configSingletonProvider(QQmlEngine *engine, QJSEngine *scriptEngine)
+{
+    Q_UNUSED(engine)
+    Q_UNUSED(scriptEngine)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_UNUSED is a macro then please configure it.

(unknownMacro)